wfl/Controllers/FoodOrdersController.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Symphono.Wfl.Controllers
 {
-    [RoutePrefix("api/FoodOrder")]
+    [RoutePrefix("api/foodorder")]
```

Comment:
I'd suggest adding a hyphen between word boundaries. [See here](https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris)